attention.py

```diff
@@ -62,12 +62,14 @@ def attention(inputs, attention_size, time_major=False, return_alphas=False):
     b_omega = tf.Variable(tf.random_normal([attention_size], stddev=0.1))
     u_omega = tf.Variable(tf.random_normal([attention_size], stddev=0.1))
 
-    # Applying fully connected layer with non-linear activation to each of the B*T timestamps;
-    #  the shape of `v` is (B,T,D)*(D,A)=(B,T,A), where A=attention_size
-    v = tf.tanh(tf.tensordot(inputs, W_omega, axes=1) + b_omega)
+    with tf.name_scope('v'):
```

Comment:
Here does more than one operations, I think for better understanding how to produce 'v', It's  reasonable to warp these ops.